emojivoto/emojivoto-web/webapp/js/components/Vote.jsx

```diff
@@ -60,61 +76,69 @@ export default class Vote extends React.Component {
     });
   }
 
+  renderLeaderboardLink() {
+    return <Link to="/leaderboard"><div className="btn btn-blue">View the leaderboard</div></Link>;
+  }
+
   render() {
     if (this.state.error) {
-      return (
-        <div className="background-500">
-        <div className="page-content container-fluid">
-          <div className="row">
-          <div className="col-md-12">
-          <h1 className="title">Uh oh.</h1>
-          <h1 className="headline">🚧</h1>
-          <p>We couldn't process your request.</p>
-          <div className="btn btn-blue"><Link to="/" onClick={this.resetState}>Select again</Link></div>
-          </div>
-          </div>
+      let errorMessage = "We couldn't process your request.";
+      if(this.state.selectedEmoji.shortcode === ":poop:") {
+        errorMessage = "The VotePoop endpoint always returns 500s for this demo."
```

Comment:
This might be a stretch, but what if we made this a bit sassier? Something like:

> For the sake of this demo, voting for 💩  always returns an error. Get your mind out of the gutter, and [pick another](#)!

Reply:
good sass!